WORKSPACE

```diff
@@ -0,0 +1,30 @@
+workspace(name = "yams")
```

Comment:
The workspace name isn't needed unless you want to lock users to it forever. Some users may want to use other convention like `com_github_jpsim_Yams`.
```suggestion
```